src/main/java/com/apruve/JsonUtil.java

```diff
@@ -24,17 +22,10 @@ public static JsonUtil getInstance() {
 	
 	public String toJson(Object o) {
 		try {
-			StringWriter buffer = new StringWriter();
-			HashMap<String, Object> jaxbProps = new HashMap<String, Object>();
-			jaxbProps.put("eclipselink.media-type", "application/json");
-			jaxbProps.put("eclipselink.json.include-root", false);
-			JAXBContext ctx = JAXBContext.newInstance("com.apruve.models", this
-					.getClass().getClassLoader(), jaxbProps);
-			Marshaller m = ctx.createMarshaller();
-			m.marshal(o, buffer);
-
-			return buffer.toString();
-		} catch (JAXBException e) {
+			ObjectMapper mapper = new ObjectMapper();
+			mapper.registerModule(new JaxbAnnotationModule());
+			return mapper.writeValueAsString(o);
+		} catch (JsonProcessingException e) {
```

Comment:
Do we really want to return an empty string here? I think throwing an exception makes a lot more sense.

Reply:
This is here more to trap the application exception thrown by Jackson (and make the method more usable by keeping callers from having to handle it) than for any functional reason.  The unit tests verify the mapping configuration of the models, so this block should never be executed in real life.